src/components/ExampleComponent/ExampleComponent.js

```diff
@@ -4,14 +4,14 @@ import { withFirebase } from "utils/Firebase";
 const ExampleComponent = ({ firebase }) => {
```

Comment:
make sure to comment this so we know it's an example LOL mb for not telling you to earlier